sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/telemetry/Telemetry.scala

```diff
@@ -244,5 +236,5 @@ private object NoOpInstrumentation extends Instrumentation {
 
   override def buildSpan(service: Service, command: ActionCommand): Option[Span] = None
 
-  override def getTracer(): Tracer = null
+  override def getTracer: Tracer = OpenTelemetry.noop().getTracer("noop")
```

Comment:
For the Noop one, it will be good if log from time to time that user is using a noops one. 

We should not log it each time, but let's say on every 100 calls to `getTracer`. If tracing is not configured, we write a log warning. As such users become aware that they are using a feature that is not enabled.

Reply:
We have such a logging at the start of telemetry but is `.debug` level. I can put info there. Would that suffice you think?

Reply:
Fine as debug when we start telemetry. We should not make it a warning at that moment. 

But we should warn the user if they call the `getTracer` method when tracing is disable. Because in that case it's an indication that they are trying to use a disabled feature.

Reply:
Btw, it can be on a follow up PR. Let's not feature creep this one.

Reply:
added https://github.com/lightbend/kalix-jvm-sdk/issues/2074